lambdas/services/search_patient_details_service.py

```diff
@@ -35,6 +44,9 @@ def handle_search_patient_request(self, nhs_number):
             logger.audit_splunk_info(
                 "Searched for patient details", {"Result": "Patient found"}
             )
+
+            self.update_auth_session_with_permitted_search(nhs_number=nhs_number)
```

Comment:
Could this be modified to:
```suggestion
            self.update_auth_session_with_permitted_search(nhs_number)
```
?

Reply:
It's a valid comment, however I think for clarity and readability we want to enforce function calls with the same pattern `function(function_param_name=parameter_value)`